Fix minor typos, grammar, and formatting errors in the DDP video series (#2197)
pytorch/tutorials: beginner_source/ddp_series_fault_tolerance.rst

In distributed training, a single process failure can
disrupt the entire training job. Since the susceptibility for failure can be higher here, making your training

> script robust is particularly important here. You might also prefer your training job to be *elastic*, i.e.,

Reviewer: ```suggestion
script robust is particularly important here. You might also prefer your training job to be *elastic*, for example,
```

Author: I agree, this is more clear! Commit incoming
